lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -768,6 +785,24 @@ def _theoretical_driving_power(self, f_corner):
         integrated over the frequency bin corresponding to the driving input."""
         return self._theoretical_driving_power_model(f_corner)
 
+    def _motion_blur(self, acquisition_time):
+        """Include effects of motion blur into the model
+
+        Parameters
+        ----------
+        acquisition_time : float
+            Acquisition time in seconds
+        """
+        new_model = copy(self)
+        new_model._calculate_power_spectral_density = motion_blur_spectrum(
+            self._calculate_power_spectral_density, acquisition_time
+        )
```

Comment:
since this is subclassing the passive model and the first half is the same would it maybe be good to use `super()` and then just extend here?

Reply:
Good point. Fixing.